src/index.js

```diff
@@ -778,4 +778,81 @@ export default class NestedList {
      */
     this.unshiftItem();
   }
+
+  /**
+   * On paste callback that is fired from Editor
+   *
+   * @param {PasteEvent} event - event with pasted data
+   */
+  onPaste(event) {
+    const list = event.detail.data;
+
+    this.data = this.pasteHandler(list);
+
+    const oldView = this.nodes.wrapper;
+    if (oldView) {
+      oldView.parentNode.replaceChild(this.render(), oldView);
+    }
+  }
+
+  /**
+   * List Tool on paste configuration
+   *
+   * @public
+   */
+  static get pasteConfig() {
+    return {
+      tags: ["OL", "UL", "LI"],
+    };
+  }
+
+  /**
+   * Handle UL, OL and LI tags paste and returns List data with nested elements.
+   *
+   * @param {HTMLUListElement|HTMLOListElement|HTMLLIElement} element
+   * @returns {ListData}
+   */
+  pasteHandler(element) {
```

Comment:
this method should be refactored using our Code Style

Reply:
> this method should be refactored using our Code Style

Where is this code style?